mitodl/open-discussions
# add podcast subscribe link frontend

#### Pre-Flight checklist



#### What are the relevant tickets?

closes #2892 

#### What's this PR do?

this just adds a little component to show the podcast subscribe links, which we added in https://github.com/mitodl/open-discussions/pull/3066.

#### How should this be manually tested?

open a podcast in the drawer. you should see a 'subscribe' button. hovering over it should reveal 

#### Screenshots (if appropriate)

![Screenshot from 2020-07-23 16-36-00](https://user-images.githubusercontent.com/6207644/88336223-9f08bc80-cd02-11ea-8b90-8099ad26b7d0.png)
![asdfasdfasdfffff](https://user-images.githubusercontent.com/6207644/88336225-9fa15300-cd02-11ea-84fc-100ec6d84d46.png)
![Screenshot from 2020-07-23 16-35-24](https://user-images.githubusercontent.com/6207644/88336226-9fa15300-cd02-11ea-842b-f0beedb01d7c.png)
